coredb-operator/tests/integration_tests.rs

```diff
@@ -56,7 +57,7 @@ mod test {
 
         // Timeout settings while waiting for an event
         let timeout_seconds_start_pod = 60;
-        let timeout_seconds_pod_ready = 10;
+        let timeout_seconds_pod_ready = 30;
```

Comment:
now that there is a readiness probe, it doesn't just mark the pod as ready instantly when it's started

Reply:
On my local, the pod is 15s old when it becomes ready.